Libs/DICOM/Core/ctkDICOMInserterWorker.h

```diff
@@ -48,7 +48,7 @@ class CTK_DICOM_CORE_EXPORT ctkDICOMInserterWorker : public ctkAbstractWorker
   void run() override;
 
   /// Cancel worker
```

Comment:
document what methods of this class may be called from other threads.

Reply:
done